cli/cmd/up/up.go

```diff
@@ -107,7 +111,10 @@ func (u *Up) loadFileAndAnswer(c *clicontext.CLIContext) (string, map[string]str
 	if err != nil {
 		return "", nil, err
 	}
-
+	if answers == nil {
+		answers = map[string]string{}
+	}
+	answers["namespace"] = c.GetSetNamespace()
```

Comment:
Can we make this all upper case so it aligns with [here](https://github.com/rancher/rio/blob/master/stacks/rio-controller-stack.yaml#L7)
```suggestion
	answers["NAMESPACE"] = c.GetSetNamespace()
```